charmcraft/commands/store/__init__.py

```diff
@@ -409,26 +413,36 @@ class ListNamesCommand(BaseCommand):
     def fill_parser(self, parser):
         """Add own parameters to the general parser."""
         self.include_format_option(parser)
+        parser.add_argument(
+            "--include-collaborations",
+            action="store_true",
+            help="Include those names that you are a collaborator of",
```

Comment:
`Include those names that you are a collaborator` -> `Include names that you are a collaborator of` 

(`Include the names you are a collaborator of` reads well too)

Reply:
Awesome, thanks! Improved.